packages/nextjs/app/builders/[address]/_components/ChallengeDetailsStatus.tsx

```diff
@@ -0,0 +1,67 @@
+import Link from "next/link";
+import { ChallengeStatus } from "./ChallengeStatus";
+import type { MappedChallenges } from "./GroupedChallenges";
+import clsx from "clsx";
```

Comment:
This could break since it's not our direct dependency, could you remove it and implement all the related logic without that lib?